test/Sentry.Tests/HubTests.cs

```diff
@@ -1016,5 +1016,56 @@ public void ResumeSession_NoPausedSession_DoesNothing()
             // Assert
             client.DidNotReceive().CaptureSession(Arg.Is<SessionUpdate>(s => s.EndStatus != null));
         }
+
+        [Theory]
+        [InlineData(SentryLevel.Error)]
+        [InlineData(SentryLevel.Fatal)]
+        public void CaptureEvent_ErroredMessageWithoutException_ClosesCurrentSpanAsInternalError(SentryLevel level)
```

Comment:
I'm not sure we should close the span. This will affect the end timestamp by setting the end of the span to the time the error happened, even if the span was still running for a long time after.

Capturing the even should include the trace information, which will help Sentry show in the span that there was a related error. The time the span finishes shouldn't be affected. I believe neither should the status. Say there was a try/catch. The user logged the error, but handled it, and had a retry logic that then succeeded. The overall span was OK, it took longer because it did more things than it needed if it had succeeded in the first place though.

Reply:
Since spans are getting closed once the Transaction is concluded, I have removed the special code that was closing the spans if none of them were sampled.